react/features/settings/components/native/ModeratorSection.tsx

```diff
@@ -72,6 +87,7 @@ const ModeratorSection = () => {
     }, [ startAudioMuted,
         startVideoMuted,
         followMeEnabled,
+        followMeRecorderEnabled,
```

Comment:
shouldn't we add setFollowMeRecorder too.

Reply:
What do you mean? Its only the state that is used in this function.

Reply:
I meant `onFollowMeRecorderToggled` actually. Sorry!